src/main/java/org/apache/commons/lang3/BitField.java

```diff
@@ -281,7 +281,7 @@ public byte setByte(final byte holder) {
     }
 
     /**
-     * Sets a boolean BitField.
+     * Sets a {@code boolean} BitField.
```

Comment:
Hi @arturobernalg 
In this case, the bit field is a `short`, not a `boolean`, right? It's the bits in the bit field that represent booleans. I think If we are going to touch Javadocs we should make sure they are correct first ;-) See the other setters as well.